test/language/class/definition/ClassMethod_restricted-properties.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2015 Caitlin Potter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*--- 
+ description: >
+    Functions created using MethodDefinition syntactic form do not
+    have own properties "caller" or "arguments", but inherit them from
+    %FunctionPrototype%.
+---*/
+
+class Class {
+  method() {}
```

Comment:
constructor is tested in the other class-related thing. I'm not sure what the spec says about `get foo()/set foo()` for this though